src/framework/standard/mod.rs

```diff
@@ -1079,6 +1073,13 @@ impl Framework for StandardFramework {
 
                         if let Some(help) = help {
                             let groups = self.groups.clone();
+                            let mut args = {
+                                let content = message.content.chars().skip(position).skip_while(|x| x.is_whitespace())
```

Comment:
Could you add a comment stating that the `Args` creation at [#1108](https://github.com/serenity-rs/serenity/blob/fe3979a45732f96f5f3dc2d84479aac02e5f6b39/src/framework/standard/mod.rs#L1108) should also be updated (even reverse) when changing this?
Because as they're now, they're identical. I also doubt that the args for a help command should be different from a normal command.